src/modules/project/pages1/projectView/views/rewards/shared/ProjectRewardForm.tsx

```diff
@@ -272,7 +272,7 @@ export const ProjectRewardForm = ({
                       setValue('estimatedAvailabilityDate', null, { shouldDirty: true })
                     }
                   }}
-                  defaultChecked={watch('preOrder')}
+                  isChecked={watch('preOrder')}
```

Comment:
@Agill-Sheron I think I changed the defaultChecked parameter inside the controlledSwitchInput. I think it should be defaultChecked only.